### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by daeglin**

### CODE REVIEW

File: backend/src/game/routes.ts [hunk 1]
```diff
@@ -70,8 +70,8 @@ router.get(['/og/results/:id', /\/og($|\/.*)/], async (ctx) => {
 
   await ctx.render('results', {
     ...res,
-    dead: formatNumber(data.results.dead),
-    cost: formatNumber(data.results.cost, true, true),
+    dead: formattingService.formatNumber(data.results.dead),
```
Comment (jnwltr) [inline backend/src/game/routes.ts:73]: Have you tested this?

File: frontend/bin/build-lang-files.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      const lang = langFile.slice(0, -5);
+      const langData = JSON.parse(readFileSync(path.join(sourceDir, langFile), 'utf8'));
+      langs[lang] = {...langs[lang], ...langData};
+    }
+  });
+});
+
+const pagesDir = path.join(argv.directory as string, 'pages');
+readdirSync(pagesDir).forEach(pageFile => {
+  const lang = pageFile.slice(-7, -5);
```
Comment (jnwltr) [inline frontend/bin/build-lang-files.ts:28]: Wouldn't `split('.')[len-1]` be better?

File: frontend/src/app/services/i18n.service.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+export class I18nService extends FormattingService {
+
+  protected milKey = _(' mil.');
+  protected bilKey = _(' mld.');
+
+  constructor(
+    translateService: TranslateService,
+  ) {
+    super(translateService);
+  }
```
Comment (jnwltr) [inline frontend/src/app/services/i18n.service.ts:18]: Is this necessary? The super-constructor should be called by default.

File: frontend/src/app/services/i18n.service.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import {marker as _} from '@biesbjerg/ngx-translate-extract-marker';
+import {TranslateService} from '@ngx-translate/core';
+import {FormattingService} from './formatting.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class I18nService extends FormattingService {
+
+  protected milKey = _(' mil.');
```
Comment (jnwltr) [inline frontend/src/app/services/i18n.service.ts:11]: Space should not be contained.

**Review by jnwltr — COMMENTED** [review]
Have you tested the built version?

File: backend/src/game/routes.ts (hunk 1 above)
Comment (JaroslavSoldat) [reply backend/src/game/routes.ts:73]: For 'cs' (hardcoded now): Moje skóre je 146 193 mrtvých a 30 mld. Kč v nákladech.
For 'en' (only test): Moje skóre je 146,193 mrtvých a Cr 30 mld. v nákladech.

File: frontend/src/app/services/i18n.service.ts (hunk 3 above)
Comment (JaroslavSoldat) [reply frontend/src/app/services/i18n.service.ts:18]: Yes. There is TranslateService injected, so we cannot use implicit constructor.

**Review by jnwltr — APPROVED** [review]

**Pull request merged**